### DISCUSSION TITLE
What import integrations would you like to include in the platform?

### DISCUSSION THREAD

**Discussion by timlrx** [cylynx/motif.gl#163] [Ideas]
Add them in a comment and upvote the import integrations you would like to see 👍

**Comment by JoshuaN-1** [comment]
Hi, 
Not sure if this is the right place to ask but I'm trying to connect Motif to a local Neo4j instance without any luck and I can't find anything online. Any guidance would be much appreciated.
Thanks,
Josh

**Reply by timlrx** [reply]
It's kind of tricky connecting the hosted demo Motif application with a local Neo4j instance due to web browser security issues. See if the following post helps: https://community.neo4j.com/t5/general-discussions/introducing-motif-graph-explorer/m-p/33139/highlight/true

The easiest way might be to run both locally or try it out using Aura. Hope that helps.

**Reply by JoshuaN-1** [reply]
Thanks for the quick response! So running locally doesn't require any special setup, just starting the instance in Neo4j Desktop and providing the correct port number, username and password?

**Reply by timlrx** [reply]
Yup, to run motif locally, you could clone the repository and run:
```
npm install --legacy-peer-deps
npm run motif:build
npm run demo:build
npm run demo:serve
```

**Reply by JoshuaN-1** [reply]
Thank you so much!

**Comment by almereyda** [comment]
PostgreSQL

**Comment by almereyda** [comment]
MariaDB

**Comment by almereyda** [comment]
SQLite files on the web

* https://phiresky.github.io/blog/2021/hosting-sqlite-databases-on-github-pages/
* https://sqlite.org/wasm/doc/trunk/about.md

**Comment by almereyda** [comment]
Low-/No-Code platforms with relational model and CRUD APIs:

- Grist
- NocoDB
- Baserow
- Budibase
- Supabase

Other "generic" graph query languages, as in https://memgraph.com/blog/graph-database-query-languages-you-should-try plus other domain-specific "inventions" like:

* https://github.com/vaticle/typeql
* https://comunica.dev/docs/query/advanced/graphql_ld/
* https://terminusdb.com/docs/explanations/document-graph-db/woql
